vcd/resource_vcd_network_isolated_v2_nsxt_test.go

```diff
@@ -0,0 +1,160 @@
+					resource.TestCheckResourceAttr("vcd_network_isolated_v2.net1", "name", t.Name()),
+					resource.TestCheckResourceAttr("vcd_network_isolated_v2.net1", "description", "updated NSX-T isolated network test"),
+					resource.TestCheckResourceAttr("vcd_network_isolated_v2.net1", "gateway", "1.1.1.1"),
+					resource.TestCheckResourceAttr("vcd_network_isolated_v2.net1", "prefix_length", "24"),
+					resource.TestCheckResourceAttr("vcd_network_isolated_v2.net1", "static_ip_pool.#", "2"),
+					resource.TestCheckTypeSetElemNestedAttrs("vcd_network_isolated_v2.net1", "static_ip_pool.*", map[string]string{
+						"start_address": "1.1.1.10",
+						"end_address":   "1.1.1.20",
+					}),
+					resource.TestCheckTypeSetElemNestedAttrs("vcd_network_isolated_v2.net1", "static_ip_pool.*", map[string]string{
+						"start_address": "1.1.1.30",
+						"end_address":   "1.1.1.40",
+					}),
+				),
+			},
+			// Check that import works
+			resource.TestStep{ // step 3
+				ResourceName:      "vcd_network_isolated_v2.net1",
+				ImportState:       true,
+				ImportStateVerify: true,
+				ImportStateIdFunc: importStateIdOrgNsxtVdcObject(testConfig, t.Name()),
+			},
+			resource.TestStep{ // step 4
+				Config: configText3,
+				Check: resource.ComposeAggregateTestCheckFunc(
+					cachedId.cacheTestResourceFieldValue("vcd_network_isolated_v2.net1", "id"),
+					resource.TestCheckResourceAttrSet("vcd_network_isolated_v2.net1", "id"),
+					resource.TestCheckResourceAttr("vcd_network_isolated_v2.net1", "name", t.Name()),
+					resource.TestCheckResourceAttr("vcd_network_isolated_v2.net1", "description", ""),
+					resource.TestCheckResourceAttr("vcd_network_isolated_v2.net1", "gateway", "1.1.1.1"),
+					resource.TestCheckResourceAttr("vcd_network_isolated_v2.net1", "prefix_length", "24"),
+					resource.TestCheckResourceAttr("vcd_network_isolated_v2.net1", "static_ip_pool.#", "0"),
+				),
+			},
+		},
+	})
+}
+
+const TestAccVcdNetworkIsolatedV2NsxtStep1 = `
+resource "vcd_network_isolated_v2" "net1" {
+  org  = "{{.Org}}"
+  vdc  = "{{.NsxtVdc}}"
+  
+  name        = "nsxt-isolated-test-initial"
+  description = "NSX-T isolated network test"
+
+  gateway = "1.1.1.1"
+  prefix_length = 24
+
+  static_ip_pool {
+	start_address = "1.1.1.10"
+	end_address = "1.1.1.20"
+  }
+}
+`
+
+const TestAccVcdNetworkIsolatedV2NsxtStep2 = `
+resource "vcd_network_isolated_v2" "net1" {
+  org  = "{{.Org}}"
+  vdc  = "{{.NsxtVdc}}"
+  
+  name        = "{{.NetworkName}}"
+  description = "updated NSX-T isolated network test"
+
+  gateway = "1.1.1.1"
+  prefix_length = 24
+
+  static_ip_pool {
+	start_address = "1.1.1.10"
+    end_address   = "1.1.1.20"
+  }
+
+  static_ip_pool {
+	start_address = "1.1.1.30"
+	end_address   = "1.1.1.40"
+  }
+}
+`
+
+const TestAccVcdNetworkIsolatedV2NsxtStep3 = `
+resource "vcd_network_isolated_v2" "net1" {
+  org  = "{{.Org}}"
+  vdc  = "{{.NsxtVdc}}"
+  
+  name        = "{{.NetworkName}}"
+
+  gateway = "1.1.1.1"
```

Comment:
needs formatting

Reply:
Tabs vs spaces happened and my editor masked it. Should be better now.